openrtb_ext/request_wrapper.go

```diff
@@ -259,7 +259,7 @@ func (rw *RequestWrapper) rebuildDeviceExt() error {
 	return nil
 }
 
-func (rw *RequestWrapper) rebuildRequestExt() error {
+func (rw *RequestWrapper) RebuildRequestExt() error {
```

Comment:
This is intended to be a helper method of the higher level rebuild request. Can you just call the main rebuild method when the request needs to be updated?

Reply:
We wanted to build only request extension  object and to avoid checking for other objects we have exported to build only required object